sqrl-planner/sqrl-planner-local/src/main/java/com/datasqrl/plan/global/DAGAssembler.java

```diff
@@ -172,10 +174,20 @@ public PhysicalDAGPlan assemble(SqrlDAG dag, Set<URL> jars, Map<String, UserDefi
         });
 
     PhysicalDAGPlan.StagePlan streamPlan = new PhysicalDAGPlan.StagePlan(streamStage, writeDAG,
-        null, jars, udfs);
+        null, jars, udfs, null);
 
-    return new PhysicalDAGPlan(ListUtils.union(List.of(streamPlan),databasePlans));
 
+    Optional<ExecutionStage> serverStage = pipeline.getStage(Type.SERVER);
```

Comment:
I think this will look cleaner if you rewrite it to create an ArrayList and then add all the plans instead of doing a bunch of unions.

Reply:
I'm just following the style that was already there. There is a lot going on in that method so I'll save it for a later refactor.